core/controllers/get_filters.py

```diff
@@ -27,11 +27,12 @@ def get_organisation_names():
 
 
 def get_funds():
-    """Returns a list of all distinct funds.
```

Comment:
@DonShelly This seems to be just updating the `/funds` endpoint? Are we updating others e.g. `/organisations`?